Quorum/apis/llms/llm_conversation.py

```diff
@@ -0,0 +1,105 @@
+from enum import StrEnum, auto
+from pathlib import Path
+
+class MessageRole(StrEnum):
+    USER = auto()
+    ASSISTANT = auto()
```

Comment:
Personally, I would change this to an indexed conversation format, something like

```
class ConversationEntry:
    question: str
    answer: str

class Conversation:
    count: int
    conversations: list[ConversationEntry]
```